client/extensions/woocommerce/woocommerce-services/lib/utils/is-equal-case-insensitive.js

```diff
@@ -0,0 +1,14 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { isString, isEqualWith } from 'lodash';
+
+export default function isEqualCaseInsensitive( value, other ) {
```

Comment:
I would rather stand behind my decision to place the function somewhere in `utils`.

> where we are sure that the address always contain string values

As most Lodash [customizers](https://lodash.com/docs/4.17.11#isEqualWith), this one does not require string-only values. If one of the values is not a string, it will fallback to the default Lodash `isEqual` behavior.

Such comparisons might be useful in other places where addresses are involved, which is why I think that placing the function within `utils` could give it better exposure.

Reply:
Fixed here: https://github.com/Automattic/wp-calypso/pull/28666